packages/components/icons/src/custom-icon/custom-icon.styles.ts

```diff
@@ -0,0 +1,27 @@
+// use negative margin to make it so that border doesnt shift content?
+import { designTokens } from '@commercetools-uikit/design-system';
+import { css } from '@emotion/react';
+import { sizeMap } from '../leading-icon/leading-icon.styles';
```

Comment:
I think we should try to avoid coupling components the better we can.

In this case, I would suggest to have the same values here.

Maybe as a follow up we can think about having some component-level design tokens, but we will be adjusting the sizes for the icons in the short term, so I would hold the new design tokens for now.